### ISSUE TITLE
What do you bring to the table?

### ISSUE THREAD

**Issue by mattross1** [infinite-communication-solutions/infinitecommunicationsolutions#13]
Reading the repository documents it seems you guys are just an advertising firm. What will set your company part from the thousands already out there, what do you guys bring to the table? In your customer segment document you state "Our company is in charge of writing persuasive and positive reviews on products we will be advertising in order to impress and satisfy the customer into buying a product they'll need." this seems to be a deceiving and dishonest business practice, not that this isn't already done in the industry. Because it is a big deal, are companies still willing to take the risk of being publicly humiliated and exposed as social media is making it easier and easier for call to action movements and boycotts.

**Comment by mhobday** [comment]
I agree it seems you are jumping into a rather large industry without setting yourself apart from the crowd much at all. If you plan to succeed in the advertising business I doubt you'll be able to do much without something unique about yourselves.
